<PAR PT> Sandra

SanDDupt opened the pull request:
"1ère version"
@ta-webpt-par

ta-webpt-par commented on starter-code/css/main.css:
```diff
@@ -6,3 +6,161 @@ Green: #00B172
+header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
```
I would have gotten rid of the
        position : fixed
because it makes the header appear when you scroll down and would have added 
        background-color: #00B172;
        color: white;

ta-webpt-par commented on starter-code/css/main.css:
```diff
@@ -6,3 +6,161 @@ Green: #00B172
+  float: right;
+  width: 60%;
+  vertical-align: middle;
+  margin-top: 38px;
```
It would be better to use the padding and maybe the flexbox

SanDDupt commented:
Merci !

Le lun. 1 oct. 2018 à 16:18, ta-webpt-par <notifications@github.com> a
écrit :

> *@ta-webpt-par* commented on this pull request.
>
> Your code is good. It's not a problem to not finish everything. You'll
> have more occasions to advance and get better ! Keep working on it !
> ------------------------------
>
> In starter-code/css/main.css
> <https://github.com/ironhack-labs/lab-css-spotify-clone/pull/729#discussion_r221625017>
> :
>
> > +  margin: 0;
> +  padding: 0;
> +  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
> +  color: 1a1a1a;
> +}
> +
> +body {
> +  width: 100%;
> +  padding: 10px;
> +  text-align: center;
> +}
> +
> +header {
> +  width: 100%;
> +  height: 100px;
> +  position: fixed;
>
> I would have gotten rid of the
> position : fixed
> because it makes the header appear when you scroll down and would have
> added
> background-color: #00B172;
> color: white;
> ------------------------------
>
> In starter-code/css/main.css
> <https://github.com/ironhack-labs/lab-css-spotify-clone/pull/729#discussion_r221625304>
> :
>
> > +  float: left;
> +  width: 30%;
> +  margin-left: 45px;
> +}
> +header img:hover {
> +  margin-left: 60px;
> +  transition-property: margin-left;
> +  transition-duration: 1.5s;
> +  transition-timing-function: linear;
> +  transition-delay: 0.3s;
> +}
> +header .navBar {
> +  float: right;
> +  width: 60%;
> +  vertical-align: middle;
> +  margin-top: 38px;
>
> It would be better to use the padding and maybe the flexbox
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ironhack-labs/lab-css-spotify-clone/pull/729#pullrequestreview-160317075>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AndeYTQQtEfkZ-9-RxE7wl04PJgamKpWks5ugiRAgaJpZM4W-tZ3>
> .
>
